apps/sap-commerce-cloud/src/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import {
+  AppExtensionSDK,
+  FieldExtensionSDK,
+  DialogExtensionSDK,
+  init,
+  locations,
+} from '@contentful/app-sdk';
+import '@contentful/forma-36-react-components/dist/styles.css';
+import '@contentful/forma-36-fcss/dist/styles.css';
+import '@contentful/forma-36-tokens/dist/css/index.css';
+import './index.css';
+import AppConfig from './AppConfig/AppConfig';
+
+import Field from './components/Field';
+import Dialog from './components/Dialog';
+
+init((sdk) => {
+  const root = document.getElementById('root');
+  const ComponentLocationSettings = [
+    {
+      location: locations.LOCATION_APP_CONFIG,
+      component: (
+        <AppConfig
+          sdk={sdk as AppExtensionSDK}
+          name="SAP Commerce App"
+          description={`
+            The sap commerce app allows editors to select commerce objects (eg: products) from their
+            commerce app and reference them inside of Contentful entries.`}
+          logo="https://www.sap.com/dam/application/shared/logos/sap-logo-svg.svg.adapt.svg/1493030643828.svg"
```

Comment:
we should probably self-host this asset in Contentful and replace this hot-linked image resource URL before listing. cc: @gavinmatt to point us to the "right" contentful space to host this

Reply:
https://images.ctfassets.net/lpjm8d10rkpy/6pMn4nHfKoOZGwFFcqaqqe/70272257dc1d2d0bbcc3ebdde13a7358/1493030643828.svg

Reply:
- [ ] use this URL for SAP logo